diff --git a/autoload/nerdtree/ui_glue.vim b/autoload/nerdtree/ui_glue.vim
index e184703..f78425d 100644
--- a/.vim/bundle/nerdtree/autoload/nerdtree/ui_glue.vim
+++ b/.vim/bundle/nerdtree/autoload/nerdtree/ui_glue.vim
@@ -230,24 +227,10 @@ endfunction
 " FUNCTION: s:deleteBookmark(bookmark) {{{1
 " Prompt the user to confirm the deletion of the selected bookmark.
 function! s:deleteBookmark(bookmark)
-    let l:message = "Delete the bookmark \"" . a:bookmark.name
-                \ . "\" from the bookmark list?"
-
-    let l:choices = "&Yes\n&No"
-
-    echo | redraw
-    let l:selection = confirm(l:message, l:choices, 1, 'Warning')
-
-    if l:selection != 1
-        call nerdtree#echo('bookmark not deleted')
-        return
-    endif
-
     try
         call a:bookmark.delete()
         silent call b:NERDTree.root.refresh()
         call b:NERDTree.render()
-        echo | redraw
     catch /^NERDTree/
         call nerdtree#echoWarning('could not remove bookmark')
     endtry
